src/main/java/no/itera/bloggingplatform/service/impl/PostServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package no.itera.bloggingplatform.service.impl;
+
+import no.itera.bloggingplatform.model.Post;
+import no.itera.bloggingplatform.repository.PostRepository;
+import no.itera.bloggingplatform.service.PostService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class PostServiceImpl implements PostService {
+
+    private PostRepository postRepository;
+
+    @Autowired
+    public PostServiceImpl(PostRepository postRepository) {
+        this.postRepository = postRepository;
+    }
+
+    @Override
+    public List<Post> listAllPosts() {
+       return postRepository.readAll();
+    }
+
+    @Override
+    public Post createNewPost(Post post) {
+        return postRepository.create(post);
```

Comment:
Treba dat pozor na to, ze `Post` entita sa sklada z viacerych komplexnych objektov (`Author`, `Category`), cize obycajne delegovanie na repozitar nie je dostatocne. PostService bude pravdepodobne musiet mat dalsie zavislosti, ktore umoznia vytvorit autora alebo kategoriu.